src/include/dns-server/dialog-main.rb

```diff
@@ -2363,6 +2201,7 @@ def confirmAbortIfChanged
     def runExpertDialog
       expert_dialogs = [
         "start_up",
+        "use_ldap",
```

Comment:
I don't think that one ldap checkbox deserve own dialog, maybe add it to basic_options?